app.py

```diff
@@ -0,0 +1,76 @@
+import os, logging
+
+from functools import wraps
+from flask import Flask, Blueprint, session, redirect, url_for, current_app, render_template, request
+from flask.sessions import SessionInterface
+from flask_session import Session
+from pathlib import Path
+
+import app_config
+from ms_identity_web import IdentityWebPython
+from ms_identity_web.adapters import FlaskContextAdapter
+from ms_identity_web.errors import NotAuthenticatedError
+from ms_identity_web.configuration import AADConfig
+
+"""
+Instructions for running the sample app. These are dev environment instructions ONLY.
+Do not run using this configuration in production.
+
+LINUX/OSX - in a terminal window, type the following:
+=======================================================
+    export FLASK_APP=app.py
+    export FLASK_ENV=development
+    export FLASK_DEBUG=1
+    export FLASK_RUN_CERT=adhoc
+    flask run
+
+WINDOWS - in a powershell window, type the following:
+====================================================
+    $env:FLASK_APP="app.py"
+    $env:FLASK_ENV="development"
+    $env:FLASK_DEBUG="1"
+    $env:FLASK_RUN_CERT="adhoc"
+    flask run
+
+You can also use "python -m flask run" instead of "flask run"
+"""
+
+def create_app(secure_client_credential=None):
+    app = Flask(__name__, root_path=Path(__file__).parent)
+    app.config.from_object(app_config) # Flask configuration
```

Comment:
>Flask configuration [](start = 41, length = 19)

nit: //Load the Flask configuration from the aad.config.json

Reply:
this one is not for AAD configs. it's used to configure the web framework itself.